demo/llama/README.md

```diff
@@ -0,0 +1,67 @@
+The code in this directory trains the Llama 7B model on [Huggingface's RedPajama dataset](https://huggingface.co/datasets/togethercomputer/RedPajama-Data-1T-Sample/tree/main). All the code in this directory is borrowed from [Lightning AI's lit-llama repo](https://github.com/Lightning-AI/lit-llama). Changes have been made in appropriate places to fetch the data from GCS instead of reading from disk.
+
+This demo has been tested on a GCE instance with `2` Nvidia `H100` GPUs.
+
+### Download the data
+
+Follow the instructions [here](https://github.com/Lightning-AI/lit-llama/blob/main/howto/train_redpajama.md#prepare-redpajama-for-training) to download the dataset. Upload the directory with `.bin` files generated by the pre-processing script to your GCS bucket.
```

Comment:
nit: Do you think it would be useful add a command or 2 to help customers move data to GCS. ?

Reply:
Done.